### PR TITLE
Add config-dir command

### PR DESCRIPTION
**Pull request by GameFuzzy**
Needs to actually be tested on the various platforms since I can currently only confirm that it works on Linux.

Additionally, the AUR repo maintainer might want to add xdg-utils as an optional dependency since it's not included in the base package. The alternative would be parsing the mimeapps lists manually but that would be a fairly messy solution.

### CODE REVIEW

File: spicetify.go [hunk 1]
```diff
@@ -124,6 +125,10 @@ func main() {
 		}
 		return
 
+	case "show-config-dir":
```
[the remark this answers is not in the document]
Comment (afonsojramos) [reply spicetify.go:128]: @GameFuzzy

**Comment by itsmeow** [comment]
Doesn't work on Windows
```
error Error opening config directory:
fatal exec: "start": executable file not found in %PATH%
```

**Comment by itsmeow** [comment]
Fixed, assuming you tested this on Linux, is anyone able to test on macOS?

**Comment by itsmeow** [comment]
Also, not sure how to add optional deps to PKGBUILD, if you know, let me know.

**Comment by ririxidev** [comment]
> Also, not sure how to add optional deps to PKGBUILD, if you know, let me know.

It will work,
`open` is command (for `finder`) to open folders/apps from terminal.

**Comment by GameFuzzy** [comment]
> Also, not sure how to add optional deps to PKGBUILD, if you know, let me know.

Should be as simple as adding something like `optdepends=('xdg-utils: open directories in default file manager')`

**Review by itsmeow — APPROVED** [review]

**Review by afonsojramos — APPROVED** [review]

**Comment by itsmeow** [comment]
Added optdepend on AUR @GameFuzzy

**Pull request merged**